nativelink-store/src/default_store_factory.rs

```diff
@@ -51,6 +52,7 @@ pub fn store_factory<'a>(
         let store: Arc<dyn StoreDriver> = match backend {
             StoreSpec::memory(spec) => MemoryStore::new(spec),
             StoreSpec::experimental_s3_store(spec) => S3Store::new(spec, SystemTime::now).await?,
+            StoreSpec::experimental_gcs_store(spec) => GCSStore::new(spec).await?,
```

Comment:
Everywhere there is a divergence from the existing store, I'd like to understand why. I'm sure there is a good reason.